unstructured_inference/inference/layout.py

```diff
@@ -192,6 +193,21 @@ def _get_image_array(self) -> Union[np.ndarray, None]:
             self.image_array = np.array(self.image)
         return self.image_array
 
+    def annotate(self, colors: Optional[Union[List[str], str]] = None) -> Image.Image:
+        """Annotates the elements on the page image."""
+        if colors is None:
+            colors = ["red" for _ in self.elements]
+        if isinstance(colors, str):
+            colors = [colors]
+        # If there aren't enough colors, just cycle through the colors a few times
```

Comment:
shouldn't go to the default color?

Reply:
We could do that, but if someone passed a colormap to get some differentiation between the bounding boxes, and then halfway through the page they switched to only red boxes, it would be a bit weird.